.github/workflows/ci-workflow.yml

```diff
@@ -4,32 +4,34 @@ on: [push, pull_request]
 
 jobs:
   job_build_nano_debug:
-    name: Build application for NanoS, X and S+
+    name: Build application for NanoS+, X, Stax, Flex
     runs-on: ubuntu-latest
     strategy:
       matrix:
```

Comment:
Here you should have at least this mandatory build CI 
https://github.com/LedgerHQ/app-boilerplate/blob/9d8c0f551c5b5c49173c352f542f1f54f187619a/.github/workflows/build_and_functional_tests.yml#L31